project_scrum/__manifest__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
add copyright headers in all py/xml/js file following example in https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py (and children directories)